xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
Y, I'm kinda in agreement with @chrisdennis here... I really don't like this silent behavior and "just" logging doesn't sound much better. Which then opens the _escaping_ of the `${}` question... Doubling the `$` ? I wouldn't want to use `\` cause you then need a way to escape `\` which is a character that seems to be too widely use (poor windows users). Doubling `$` seems less intrusive...